tests/unit/src/presenters/question-to-inset.test.js

```diff
@@ -4,42 +4,82 @@ const moment = require("moment");
 describe("question-to-inset", () => {
   let translate;
   let question;
-  let locals;
+  let data;
   const language = "en";
 
   beforeEach(() => {
     question = {
       questionKey: "rti-payslip-national-insurance",
+      info: {
+        months: 3,
+      },
     };
-    locals = {
+    data = {
       dynamicDate: moment()
-        .subtract(3, "months")
+        .subtract(question.info.months, "months")
         .locale(language)
         .format("DD MMMM YYYY"),
     };
 
     translate = jest.fn();
   });
 
+  it("should call translate using questionID and months", () => {
+    presenters.questionToInset(question, translate, language);
+
+    expect(translate).toHaveBeenCalledWith(
+      "fields.rti-payslip-national-insurance.inset",
+      data
+    );
+  });
+
+  it("should call translate using questionID and 12 months", () => {
+    question.info.months = 12;
+    data = {
+      dynamicDate: moment()
+        .subtract(question.info.months, "months")
+        .locale(language)
+        .format("DD MMMM YYYY"),
+    };
+
+    presenters.questionToInset(question, translate, language);
+
+    expect(translate).toHaveBeenCalledWith(
+      "fields.rti-payslip-national-insurance.inset",
+      data
+    );
+  });
+
+  it("should call translate using questionID without month when month is 0", () => {
```

Comment:
`when month is undefined` ? (we won't expect it to be provided at all if not needed)